src/ngx_http_lua_ssl.c

```diff
@@ -34,4 +38,160 @@ ngx_http_lua_ssl_init(ngx_log_t *log)
 }
 
 
+static size_t
+ngx_http_lua_ssl_get_error(u_long e, u_char *ssl_err_buf,
+    size_t ssl_err_buf_len, u_char *default_errmsg,
+    size_t default_errmsg_len)
+{
+    size_t len;
+
+    if (e == 0) {
+        len = ngx_min(ssl_err_buf_len, default_errmsg_len);
+        ssl_err_buf = ngx_copy(ssl_err_buf, default_errmsg, len);
```

Comment:
The return value is useless. Better use `ngx_memcpy` here instead of `ngx_copy`.